### PR TITLE
Disable GCE external load balancer when services handled by Ingress-GCE

### PR DESCRIPTION
**Pull request by panslava**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:

Add opt-in annotation for Regional Backend Service mode in GCE External LoadBalancer. If annotation is enabled, k/k will not handle LoadBalancer  

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

**Comment by panslava** [comment]
> 

Added CLA

**Comment by cezarygerard** [comment]
@cheftako 
Could you please take a look a this PR which provider-specific code from the Service Controllers

Regards
Cezary

File: staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_external.go [hunk 1]
```diff
@@ -51,6 +54,19 @@ const (
...
+	// Skip service handling if managed by ingress-gce using Regional Backend Services
+	if val, ok := apiService.Annotations[RBSAnnotationKey]; ok && val == RBSEnabled {
+		return nil, cloudprovider.ImplementedElsewhere
+	}
+	// Skip service handling if service has Regional Backend Services finalizer
+	if hasFinalizer(apiService, ELBRbsFinalizer) {
+		return nil, cloudprovider.ImplementedElsewhere
+	}
+	// Skip service handling if it has Regional Backend Service created by Ingress-GCE
+	if existingFwdRule.BackendService != "" {
```
Comment (kl52752) [inline staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_external.go:66]: you need to check if fwd rule exist
if existingFwdRule != nil &&  existingFwdRule.BackendService != ""

Any tests for this case?

**Pull request merged**